clang/test/CodeGenOpenCL/builtins-fp-atomics-gfx12.cl

```diff
@@ -48,7 +48,7 @@ void test_local_add_2f16_noret(__local half2 *addr, half2 x) {
 }
 
 // CHECK-LABEL: test_flat_add_2f16
-// CHECK: [[RMW:%.+]] = atomicrmw fadd ptr %{{.+}}, <2 x half> %{{.+}} syncscope("agent") seq_cst, align 4, !amdgpu.no.fine.grained.memory !{{[0-9]+$}}
+// CHECK: [[RMW:%.+]] = atomicrmw fadd ptr %{{.+}}, <2 x half> %{{.+}} syncscope("agent") monotonic, align 4, !amdgpu.no.fine.grained.memory !{{[0-9]+$}}
```

Comment:
why was this changed?

Reply:
I'm not sure. My guess is this changed in the downstream patch change to seq_cst to monotonic, and this diff wasn't updated? Before this patch the atomicrmw wouldn't have been produced at all

Reply:
it passes BuildBot tests